code/modules/mob/living/carbon/alien/humanoid/emote.dm

```diff
@@ -60,6 +60,26 @@
 				message = text("<span class='name'>[src]</span> signs[].", (text2num(param) ? text(" the number []", text2num(param)) : null))
 				m_type = 1
 
+		if ("special")	//visual emotes with their own special sprites - DO NOT BE BLIND LIKE A CERTAIN SOMEONE WHO SHOULD HAVE ASKED WHAT EACH SPRITE WAS
+			if (sprite_changed_for_emote)
+				src << "You are currently brimming with emotion!"
+			else
+				sprite_changed_for_emote = TRUE	//so we don't attempt to swap the sprite twice in proc/change_icon because I don't know what may happen if we do
+				if (caste == "h")	//for hunter xenos

```

Comment:
Can it be changed to a define? caste is a var in humanoid without value and it's set in the... castes of aliens that inherit from it.